velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -1079,9 +1102,14 @@ void assertResults(
   }
 
   auto expectedRows = duckDbQueryRunner.execute(duckDbSql, resultType);
+  const auto expectedType = (expectedRows.size() == 0) ? nullptr : resultType;
```

Comment:
We can always pass the `expectedType` and `actualType` as is here.

Reply:
Fixed.

Reply:
`actualType` cannot be passed as such as the `results` vector for an empty resultset could be 0.